client/my-sites/plans/current-plan/current-plan-thank-you/search-thank-you.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import ThankYou from './thank-you';
+
+const SearchProductThankYou = ( { translate } ) => (
+	<ThankYou
+		illustration="/calypso/images/illustrations/security.svg"
+		showContinueButton
+		title={ translate( 'Hello Jetpack Search!' ) }
+	>
+		<p>{ translate( 'We just finished setting up search for you.' ) }</p>
+		<p>
+			{ translate(
+				'Next, we’ll take a look at your new WordPress.com dashboard. ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Next, we’ll take a look at your new WordPress.com dashboard. You can manage your backups under “Tools > Activity” in the sidebar. There’s also a checklist to help you get the most out of your Jetpack plan.' )` *ES Score: 9*